config/assisted-service/kustomization.yaml

```diff
@@ -1,10 +1,10 @@
 resources:
 - scality-secret.yaml
 - scality-public-secret.yaml
-- postgres-deployment.yaml
-- assisted-service.yaml
+#- postgres-deployment.yaml
```

Comment:
why is that commented out?

Reply:
postgres is deployed in the assisted-service pod we create.